libs/server/ACL/ACLParser.cs

```diff
@@ -97,28 +97,28 @@ public static void ApplyACLOpToUser(ref User user, string op)
                 return;
             }
 
-            if (op == "on")
+            if (op.Equals("on", StringComparison.Ordinal))
```

Comment:
I think we should use `OrdinalIgnoreCase` for all ACL op keywords to improve robustness (`on`, `off`, `nopass`, `reset`, `resetpass`, `allkeys `and `resetkeys`).